tinygrad/tensor.py

```diff
@@ -316,8 +316,8 @@ def num_zeros(step, dim_sz): return 0 if step == 1 or (y := dim_sz % step) == 0
         dim_shape = next(it_shape)
         if isinstance(s, slice): final_shape.append(dim_shape)
         elif tensor_found:
-          for i_ in range(len(tensor_found)):
-            if tensor_found[i_][0] > i: sub[i_] -= 1
```

Comment:
Could probably revert the change here, because `__getitem__` is refactored in #1661

Reply:
@wozeparrot Done.